CHANGELOG.md

```diff
@@ -1,3 +1,86 @@
+## [4.0.0-dev.1](https://github.com/VenusProtocol/venus-protocol/compare/v3.1.0...v4.0.0-dev.1) (2023-09-25)
+
+
+### ⚠ BREAKING CHANGES
+
+* Removal of comptroller.sol
+
+### Features
+
+* diamond proxy implementation of comptroller ([66f90f4](https://github.com/VenusProtocol/venus-protocol/commit/66f90f4564e73993b7885c4bbd15c8d9d7d74437))
+* force liquidation implementation ([cf9c7cb](https://github.com/VenusProtocol/venus-protocol/commit/cf9c7cb74519ae8fc8c99add9997f63087ac2f96))
+* ven-1619 3.4 added view methods for facets states ([d12981f](https://github.com/VenusProtocol/venus-protocol/commit/d12981f2cbb3962d170d0348ede45a07457419b5))
```

Comment:
Let us remove this feature point as we released these changes after introducing them.

Reply:
It's automatically generated, I would edit this file